inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -195,10 +195,8 @@
                         "type": "string"
                     },
                     "orcid": {
-                        "description": "ORCID Id when available",
                         "format": "orcid",
-                        "pattern": "\\d{4}-\\d{4}-\\d{4}-\\d{4}",
-                        "title": "ORCID ID",
```

Comment:
Why removing the other properties? (seems unrelated to the untrained eye, aka, me :wink: )